src/datepicker/datepicker.js

```diff
@@ -104,6 +104,16 @@ export default class Datepicker<T = Date> extends React.Component<
           } else {
             nextDate = [start, start];
           }
+        } else if (
+          this.dateHelpers.dateRangeIncludesDates(
+            // $FlowFixMe Cannot call `this.dateHelpers.dateRangeIncludesDates` with `nextDate` bound to the first parameter because  read-only array type [1] is incompatible with  array type [2]
```

Comment:
It thinks `DateValueT` is a read-only array, but the actual type is `DateValueT<T> = ?T | Array<?T>`

Reply:
I did a bunch of testing to see why this is erroring but no luck :(